Mypy error: `Missing positional argument "version_id" in call to "__init__"  [call-arg]`

I don't know what is the proper fix here, but `GazeModel` is broken.

```
roboflow/models/gaze.py:16: error: Missing positional argument "version_id" in call to "__init__"  [call-arg]
```

--- a/roboflow/models/gaze.py
+++ b/roboflow/models/gaze.py
@@ -6,11 +6,12 @@ class GazeModel(InferenceModel):
     Run inference on a gaze detection model, hosted on Roboflow.
     """
 
-    def __init__(self, api_key: str):
+    def __init__(self, api_key: str, version_id: str):
         """
         Initialize a CLIP model.
 
         Args:
             api_key: Your Roboflow API key.
+            version_id (str): the ID of the dataset version to use for inference
         """
-        super().__init__(api_key=api_key)
+        super().__init__(api_key=api_key, version_id=version_id)

